packages/client/hmi-client/src/stores/auth.ts

```diff
@@ -92,7 +92,7 @@ const useAuthStore = defineStore('auth', {
 			window.location.assign('/logout');
 		},
 		autoRenew() {
-			console.log('RENEW SSO');
+			console.info('RENEW SSO');
```

Comment:
Is this meant to stay `console` or should it be logger?

Reply:
This is only for the front-end to now that the SSO has been renewed.
I don't think we need to call it anymore. @bigglesandginger what do you think?